docs/config.rst

```diff
@@ -6,10 +6,20 @@ To get started, you need to configure Raven to use your Sentry DSN:
 .. sourcecode:: javascript
 
     var Raven = require('raven');
-    Raven.config('___PUBLIC_DSN___').install()
+    Raven.config('___PUBLIC_DSN___').install();
 
 At this point, Raven is ready to capture any uncaught exceptions.
 
+``install`` method can optionally take a callback function, in case a fatal error occurs,
+which means that something during capturing initial error went wrong and process is about to exit.
+
+.. sourcecode:: javascript
+
+    Raven.config('___PUBLIC_DSN___').install(function (err, initialErr, eventId) {
+      console.error(err);
+      process.exit(1);
+    });
+
```

Comment:
Some small language tweaks I'd recommend:

> Note that the ``install`` method can optionally take a callback function that is invoked if a fatal, non-recoverable error occurs.  You can use this callback to perform any cleanup before the Node process exits.
 +which means that something during capturing initial error went wrong and process is about to exit.